haskell/ghc_bindist.bzl

```diff
@@ -107,42 +111,51 @@ ghc_bindist = repository_rule(
             values = _GHC_BINS.keys(),
             doc = "The desired GHC version",
         ),
+        "target": attr.string(),
     },
 )
-"""Create a new repository from binary distributions of GHC. The
-repository exports two targets:
-
-* a `bin` filegroup containing all GHC commands,
-* a `threaded-rts` CC library.
-
-These targets are unpacked from a binary distribution specific to your
-platform. Only the platforms that have a "binary package" on the GHC
-[download page](https://www.haskell.org/ghc/) are supported.
-
-Example:
-   In `WORKSPACE` file:
-
-   ```bzl
-   load("@io_tweag_rules_haskell//haskell:haskell.bzl", "ghc_bindist")
-
-   # This repository rule creates @ghc repository.
-   ghc_bindist(
-     name    = "ghc",
-     version = "8.2.2",
-   )
-
-   # Register the toolchain defined locally in BUILD file:
-   register_toolchains("//:ghc")
-   ```
-
-   In `BUILD` file:
-
-   ```bzl
-   # Use binaries from @ghc//:bin to define //:ghc toolchain.
-   haskell_toolchain(
-     name = "ghc",
-     version = "8.2.2",
-     tools = "@ghc//:bin",
-   )
-   ```
-"""
+
+def ghc_bindist(name, version, target):
+    """Create a new repository from binary distributions of GHC. The
+    repository exports two targets:
+
+    * a `bin` filegroup containing all GHC commands,
+    * a `threaded-rts` CC library.
+
+    These targets are unpacked from a binary distribution specific to your
+    platform. Only the platforms that have a "binary package" on the GHC
+    [download page](https://www.haskell.org/ghc/) are supported.
+
+    Example:
+       In `WORKSPACE` file:
+
+       ```bzl
+       load("@io_tweag_rules_haskell//haskell:haskell.bzl", "ghc_bindist")
+
+       # This repository rule creates @ghc repository.
+       ghc_bindist(
+         name    = "ghc",
+         version = "8.2.2",
+       )
+
+       # Register the toolchain defined locally in BUILD file:
+       register_toolchains("//:ghc")
```

Comment:
The docs seem outdated. As I understand the changes, this should no longer be required of the user.

Reply:
(resolved in later commit)